Графика для всех страниц

va1erymakar0v opened the pull request:
v1.1

- Исправил замечания
- Добавил пропущенный скрытый заголовок в блок социальных сетей в начале страницы
- Немного улучшил логику именования классов в карточках продуктов

---
:mortar_board: [Разметка внутренней страницы](https://up.htmlacademy.ru/htmlcss/33/user/1945443/tasks/4)

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/33/user/1945443/tasks/5)

va1erymakar0v committed:
Исправляет замечания в разметке index.html

v1.1

- Исправил замечания

- Добавил пропущенный скрытый заголовок в блок социальных сетей в начале страницы

- Немного улучшил логику именования классов в карточках продуктов

va1erymakar0v committed:
Добавляет страницу catalog.html

v1.0

1kazakov commented on catalog.html:
```diff
@@ -0,0 +1,387 @@
+
+        <ul class="navigation__list navigation-actions">
+          <li class="navigation__item">
+            <a class="navigation__phone" href="tel:+78005558628">+7 800 555-86-28</a>
```
еще раз подумал над телефоном и мне кажется его лучше вынести из этого списка и оставить отдельно как логотип

1kazakov commented on catalog.html:
```diff
@@ -0,0 +1,387 @@
+            </button>
+          </li>
+
+          <li class="product creamy">
```
про карточку товара, тоже сразу не досмотрел, если посмотреть карточки товаров во многих интернет магазинах то при нажатии на нее происходит переход на страницу с этим товаром то есть у нас должна быть ссылка

1kazakov commented on catalog.html:
```diff
@@ -0,0 +1,387 @@
+            <h3 class="product__name">Фисташка</h3>
+            <span class="product__description">Фисташковый пломбир с кусочками шоколада</span>
+            <b class="product__price">340 ₽/кг</b>
+            <button class="product__buy-button" type="button">
```
и эта кнопка на самом деле не кнопка, она тоже должна вести на карточку товара где будет возможность выбрать обьем покупаемого мороженного и еще что то

va1erymakar0v committed:
Исправляет замечания по разметке страниц

index - v1.2, catalog - v1.1

- телефон в хэдэре сделан отдельным элементом
-карточки товаров и кнопка покупки сделаны ссылками

va1erymakar0v committed:
Добавляет папку с графикой
